pyhive/sqlalchemy_hive.py

```diff
@@ -296,7 +312,7 @@ def _get_table_columns(self, connection, table_name, schema):
                 raise exc.NoSuchTableError(full_table)
             return rows
 
-    def has_table(self, connection, table_name, schema=None):
+    def has_table(self, connection, table_name, schema=None, **kw):
```

Comment:
https://docs.sqlalchemy.org/en/20/core/internals.html#sqlalchemy.engine.Dialect.has_table

Without `**kw` one of the test cases was failing with 2.0.